words/tools.asm

```diff
@@ -484,3 +515,86 @@ _words_done:
                 inx
 
 z_words:        rts
+
+
+
+; djb2 16-bit hash algorithm
+;    v = 5381
+;    for c in s:
+;        v = (v*33 + c) mod 1<<16
+
+xt_djb2:
```

Comment:
hmm, this is extraneous, I'll move it back to my local collection of extra words.

btw are you interested in having something like `words\extras.asm` with contributions like this?  there are a few things I've found useful like a simple random number generator etc.